src/controllers/userContoller.js

```diff
@@ -0,0 +1,103 @@
+const {
+  getUsers,
+  addUser,
+  getUser,
+  deleteUser,
+  updateUser,
+} = require('../services/users.service');
```

Comment:
There is a typo in the file name. It should be `userController.js` instead of `userContoller.js`. This typo should be corrected to ensure consistency and avoid confusion.